include/cantera/equil/ChemEquil.h

```diff
@@ -238,7 +238,13 @@ class ChemEquil
      * Given a vector of dimensionless element abundances, this routine
      * calculates the moles of the elements and the moles of the species.
      *
-     * @param[in] x = current dimensionless element potentials..
+     * @param s ThermoPhase object
+     * @param[in] x current dimensionless element potentials
+     * @param n_t
+     * @param[in] Xmol_i_calc Mole fractions of the species
+     * @param eMolesCalc
+     * @param n_i_calc
+     * @param[in] pressureConst Pressure
```

Comment:
While this change makes the warning go away, the underlying problem -- lack of documentation for several arguments -- still exists. I'd rather have the warning as a reminder that this ought to be properly fixed than just suppressing the warning.